paddle/scripts/paddle_build.sh

```diff
@@ -722,6 +722,32 @@ function fetch_upstream_develop_if_not_exist() {
     fi
 }
 
+function check_whl_size() {
+    if [ ! "${pr_whl_size}" ];then
+        echo "pr whl size not found "         
+        exit 1
+    fi
+
+    set +x
+    dev_whl_size=`du -m ${PADDLE_ROOT}/build/python/dist/*.whl|awk '{print $1}'`
```

Comment:
Seems like we're trying to get "dev_whl_size" and "pr_whl_size" from the same place: ${PADDLE_ROOT}/build/python/dist/*.whl

Is it because we rebuilt "develop" branch after obtaining "pr_whl_size" ?

Reply:
是的，会执行两边，第一次是PR的，第二次是develop分支的